TrackingTools/TrajectoryState/BuildFile.xml

```diff
@@ -6,11 +6,11 @@
 <use   name="DataFormats/TrackReco"/>
 <use   name="DataFormats/TrajectoryState"/>
 <use   name="DataFormats/BeamSpot"/>
+<use   name="MagneticField/Engine"/>
 <use   name="TrackingTools/AnalyticalJacobians"/>
 <use   name="TrackingTools/TrajectoryParametrization"/>
-<use   name="MagneticField/Engine"/>
 
-<use   name="vdt_headers"/>
+<use   name="vdt"/>
```

Comment:
This was done a couple of days ago. Please go ahead with your changes.
On Fri 13 Mar 2015 at 18:34 wmtan <notifications@github.com> wrote:

> In TrackingTools/TrajectoryState/BuildFile.xml
> <https://github.com/cms-sw/cmssw/pull/8094#discussion_r26405939>:
>
> >
> > -<use   name="vdt_headers"/>
> > +<use   name="vdt"/>
>
> Do not make this change. The 7_4_X dependency is the correct one, and I
> already have a PR that changes the 7_4_ROOT6_X version of this file to the
> 7_4_X version
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/8094/files#r26405939>.
>